app/presenters/two_factor_login_options_presenter.rb

```diff
@@ -78,7 +78,7 @@ def account_reset_link
       'two_factor_authentication.account_reset.text_html',
       link: @view.link_to(
         t('two_factor_authentication.account_reset.link'),
-        account_reset_request_path(locale: LinkLocaleResolver.locale),
+        account_reset_recovery_options_url(locale: LinkLocaleResolver.locale),
```

Comment:
I think we'll need to put this behind a feature flag to avoid backwards incompatibility in the 50/50 deploy state.  If I a server on the new version redirects me to the new route, but the request is handled by the old version, I'll get a 404.